czishrink/netczicompressTests/Models/ProgramNameAndVersionTests.cs

```diff
@@ -18,8 +18,7 @@ public void ToString_WhenCalled_ReturnsExpected()
         var actual = new ProgramNameAndVersion().ToString();
 
         // ASSERT
-        var re = new Regex(@"^CZI Shrink 1\.0\.0(\+\d+)?$");
-        re.IsMatch(actual).Should().BeTrue();
+        actual.Should().MatchRegex(@"^CZI Shrink 1\.0\.\d+(\+\d+)?$");
```

Comment:
What about using `@"^CZI Shrink [1-9]+\.[0-9]+\.\d+(\+\d+)?$"` to avoid having to update test for minor/major updates?

Reply:
Decided to keep as is by majority vote.